string_utility.c

```diff
@@ -34,3 +34,26 @@ void _puts(char *s)
 		len = write(1, s, len);
 	return (len);
 }
+
+/**
+ * _strcmp - compare two string
+ * @s1: input string
+ * @s2: input string
+ *
+ * Return: s1[i] - s2[i]
+ */
+
+int _strcmp(char *s1, char *s2)
+{
+	int i = 0;
+
+	while (s1[i] != '\0' && s2[i] != '\0')
+	{
+		if (s1[i] != s2[i])
+		{
+			return (s1[i] - s2[i]);
+		}
+		i++;
+	}
+	return (0);
```

Comment:
here you need to return `(s1[i] - s2[i])`, testing the case of comparing "toto" with "toto." your code will return 0, but the strings are not identical.

Reply:
didn't pay attention to it, return value changed at commit 9aab630